Step07/app/src/test/java/net/agilepartner/workshops/cqrs/core/MyAggregate.java

```diff
@@ -18,12 +18,14 @@ public void changeName(String name) {
         }
     }
 
-    @SuppressWarnings("unused")
-    private void apply(NameChanged evt) {
-        name = evt.name;
+    @Override
```

Comment:
Same thing here. You lose expressiveness of your domain code just because you want to make your base class simpler. I wouldn't to that, and instead, find a way to make the base class cleaner without impairing the clean domain.

Reply:
Same as for ```InventoryItem```